src/forwarder.ts

```diff
@@ -34,14 +34,17 @@ function generateSubscriptionFilterName(functionUniqueId: string, forwarderArn:
   return subscriptionFilterName;
 }
 
-export function addForwarder(scope: cdk.Construct, lambdaFunctions: lambda.Function[], forwarderArn: string) {
+function getForwarder(scope: cdk.Construct, forwarderArn: string) {
```

Comment:
I like how you moved the forwarder stuff into a separate function, it keeps the code cleaner and easier to read